compassion_dashboard/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- encoding: utf-8 -*-
```

Comment:
The new standard is `coding` instead of `encoding`. Please change all your headers.